Friends database table doesn't exist.

Probably just an issue when jumping around in development versions, but we need to make sure that this isn't an issue for a release:

The table "friends" isn't created if the user already has an older version.

We need to make sure the DB version is checked and code is run to create the table on upgrade. The

```
E/AndroidRuntime( 1764): FATAL EXCEPTION: AsyncTask #1
E/AndroidRuntime( 1764): java.lang.RuntimeException: An error occured while executing doInBackground()
E/AndroidRuntime( 1764):    at android.os.AsyncTask$3.done(AsyncTask.java:299)
E/AndroidRuntime( 1764):    at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:352)
E/AndroidRuntime( 1764):    at java.util.concurrent.FutureTask.setException(FutureTask.java:219)
E/AndroidRuntime( 1764):    at java.util.concurrent.FutureTask.run(FutureTask.java:239)
E/AndroidRuntime( 1764):    at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:230)
E/AndroidRuntime( 1764):    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1080)
E/AndroidRuntime( 1764):    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:573)
E/AndroidRuntime( 1764):    at java.lang.Thread.run(Thread.java:841)
E/AndroidRuntime( 1764): Caused by: android.database.sqlite.SQLiteException: no such table: friends (code 1): , while compiling: create table temp_table as select * from friends
```

--- a/Atarashii/src/net/somethingdreadful/MAL/sql/MALSqlHelper.java
+++ b/Atarashii/src/net/somethingdreadful/MAL/sql/MALSqlHelper.java
@@ -162,19 +162,10 @@ public class MALSqlHelper extends SQLiteOpenHelper {
             db.execSQL("insert into " + TABLE_MANGA + " select * from temp_table;");
             db.execSQL("drop table temp_table;");
         }
-        
-        if (oldVersion < 6) {            
-        	db.execSQL("create table temp_table as select * from " + TABLE_FRIENDS);
-            db.execSQL("drop table " + TABLE_FRIENDS);
+
+        if (oldVersion < 6) {
             db.execSQL(CREATE_FRIENDS_TABLE);
-            db.execSQL("insert into " + TABLE_FRIENDS + " select * from temp_table;");
-            db.execSQL("drop table temp_table;");
-            
-            db.execSQL("create table temp_table as select * from " + TABLE_PROFILE);
-            db.execSQL("drop table " + TABLE_PROFILE);
             db.execSQL(CREATE_PROFILE_TABLE);
-            db.execSQL("insert into " + TABLE_PROFILE + " select * from temp_table;");
-            db.execSQL("drop table temp_table;");
         }
     }
 }

